Networking/Networking/Model/Copiable/Models+Copiable.generated.swift

```diff
@@ -169,6 +169,30 @@ extension CouponReport {
     }
 }
 
+extension DotComUser {
```

Comment:
super nit: I think the existing `Dotcom*` structs have `com` as lowercase, super minor though